compile.sh

```diff
@@ -14,6 +14,13 @@
 # http://www.armbian.com/using-armbian-tools/
 # for detailed explanation of these parameters
 
+function do_clean_up()
+{
+	chown -R $(/usr/bin/logname):$(/usr/bin/id -g $(/usr/bin/logname)) $SRC
```

Comment:
Don't know if this is a good idea. Maybe limit this to $SRC/lib only if you have problems with git clones via SSH?